### ISSUE TITLE
Plugin's not on the community gallery

### ISSUE THREAD

**Issue by ryanjamurphy** [akaalias/obsidian-shuffle#4]
Post it, @akaalias!

**Comment by Moonbase59** [comment]
Yeah, please! Just found it mentioned in one of Eleanor’s videos, and was so disappointed that it couldn’t be installed from the community gallery.
